src/test/java/ConfigLoader.java

```diff
@@ -0,0 +1,31 @@
+import java.io.FileNotFoundException;
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+public class ConfigLoader {
+    private InputStream inputStream;
+    private Properties prop;
+
+    public ConfigLoader() throws IOException {
```

Comment:
Get familiar with Singleton pattern and rewrite ConfigLoader using Singleton pattern.
https://gist.github.com/eliasnogueira/df8edc58c792d176c1f0c730cd73f054
But use try with resources here.